tron/api/requestargs.py

```diff
@@ -13,20 +13,29 @@ def get_integer(request, key):
     """Returns the first value in the request args for the given key, if that
     value is an integer. Otherwise returns None.
     """
-    if not request.args or key not in request.args:
+    value = get_string(request, key)
+    if value is None or not value.isdigit():
         return None
 
-    value = request.args[key][0]
-    if not value.isdigit():
-        return None
     return int(value)
 
 
 def get_string(request, key):
     """Returns the first value in the request args for a given key."""
-    if not request.args or key not in request.args:
+    if not request.args:
+        return None
+
+    if key is not bytes:
+        key = key.encode()
+
+    if key not in request.args:
```

Comment:
You can check if key is not bytes first. So this can be something like:
```
    if not request.args or key not in request.args:
        return None
```
I guess it might be easier to read?